src/components/search-package-list-item/search-package-list-item.js

```diff
@@ -108,6 +112,10 @@ SearchPackageListItem.propTypes = {
   packageName: PropTypes.string,
 };
 
+SearchPackageListItem.propTypes = {
```

Comment:
I believe this should be `defaultProps`, right?

Reply:
Yep, you're right, misprint